doc/manual/source/language/evaluation.md

```diff
@@ -0,0 +1,77 @@
+# Evaluation
+
+Evaluation is the process of turning a Nix expression into a [Nix value](types.md).
+
+This happens by a number of rules, such as:
+- Constructing values from literals. 
+  For example the number literal `1` is turned into the number value `1`.
```

Comment:
This will render inline, which is not great for readability. Should be a paragraph under the list item.

For some reasone I can't check https://nix.dev/manual/nix/development/ atm because the anubis screen doesn't ever go away (@NixOS/infra any clue what could be going on there?)

Also generally these examples should ideally be expression-value-pairs like in the Nix language tutorial: https://nix.dev/tutorials/nix-language#notes-on-whitespace. Those sentences contain mostly visual noise for what could be very few characters. We had a discussion a long time ago on encoding that structure in the documentation source, but it seems to have led nowhere.

Reply:
Yes it does

Reply:
Can we consume that differently, e.g. by fetching and hosting the built documentation?

Reply:
Temporarily reverted Anubis until we have this sorted.